ui/app/templates/components/fs/file.hbs

```diff
@@ -32,7 +32,7 @@
       <h3 class="empty-message-headline">Unsupported File Type</h3>
       <p class="empty-message-body message">The Nomad UI could not render this file, but you can still view the file directly.</p>
       <p class="empty-message-body">
-        <a data-test-log-action="raw" class="button is-light" href="{{this.catUrl}}" target="_blank" rel="noopener noreferrer">View Raw File</a>
+        <a data-test-log-action="raw" class="button is-light" onclick={{action "downloadFile"}} target="_blank" rel="noopener noreferrer">View Raw File</a>
```

Comment:
Since this is no longer a link, it should be a button for semantic and accessibility reasons.

```suggestion
        <button data-test-log-action="raw" class="button is-light" onclick={{action "downloadFile"}}>View Raw File</button>
```

Reply:
Thanks for review!
I fixed issues with tag, added same to other View Raw File buttons, and tried to solve problem with tests - add check that proper url was called.

----
That about cookie - I tried to look that way, it was my first thought, but I haven't found any way to simply implement that. It would be much better solution than this one.

About leaving <a> if there is no ACL - it a bit hard to do - because there should be specific anonymous ACL to allow file downloads (currently I use this on our server to allow other users to download files through UI). And I don't see the way to determine is this ACL available or not from UI side.